cli/Cargo.toml

```diff
@@ -22,6 +22,7 @@ percent-encoding = "^2.1.0"
 
 [dev-dependencies]
 async-std = { version = "1.12.0", features = ["attributes", "tokio1"] }
+uuid = "1.3.0"
```

Comment:
Version 1.3.3 is out

Reply:
Cool. I think we should do a separate deps bump PR for updating all dependencies. yeah?